x-pack/filebeat/module/sophos/xg/test/event.log-expected.json

```diff
@@ -60,7 +60,7 @@
         "@timestamp": "2020-05-18T14:38:58.000-02:00",
         "client.ip": "83.20.132.250",
         "destination.as.number": 721,
-        "destination.as.organization.name": "DoD Network Information Center",
+        "destination.as.organization.name": "DNIC-ASBLK-00721-00726",
```

Comment:
This looks like a big change to the expected values. It's very likely to affect users that are filtering or alerting on AS org names. Do we have any idea if this is a change caused by something in Elasticsearch code or if it's something that MaxMind changed?

@probakowski Would you happen to have any insight? (saw you on a bunch of geoip related PRs)

Reply:
All changes for `*.geo.*` fields  are for sure connected with GeoIPv2 project that finished recently. We haven't updated MaxMind databases for a long time (end of 2019?) due to license change and now we do update them to the latest released by MaxMind every 3 days (we still ship old databases but right after cluster start this new component kicks in and downloads new files)